webcurator-webapp/build.gradle

```diff
@@ -61,7 +59,7 @@ dependencies {
     providedRuntime 'org.springframework.boot:spring-boot-starter-tomcat'
 
     implementation 'com.h2database:h2:1.4.199'
-    implementation 'mysql:mysql-connector-java:5.1.6'
+    implementation 'mysql:mysql-connector-java:8.0.21'
```

Comment:
What versions of mysql did you test this driver (8.0.21) with? Is it backwards compatible with older versions?